sessions/recap-project-6-NEW/diy-hub/user-stories/1_projects-list.md

```diff
@@ -0,0 +1,34 @@
+# Projects List
+
+## Value Proposition
+
+**As a user**
+
+**I want to** browse a list of DIY projects,
+
+**so that** I can easily find and explore new projects that interest me.
+
+## Description
+
+![wireframe](./assets/projects-list.png)
```

Comment:
I think the wireframes are fine, but maybe we want to choose a different layout for the projects, otherwise they might look too similar in the end? We could choose a one column layout with the information to the side of the image. Totally optional, just wanted to share my thoughts on this. Please decide for yourself.